### PR TITLE
docs: add faq

### LINKED ISSUE #44170: 文件夹上传组件在safria的兼容性问题 在选择“上传文件夹”上传文件的时候 正常应该不能选文件，但现在能选文件
### Reproduction link

[https://ant.design/components/upload-cn#components-upload-demo-directory](https://ant.design/components/upload-cn#components-upload-demo-directory)

### Steps to reproduce

点击上传文件夹组件，

### What is expected?

上传文件的时候 正常应该不能选文件，但现在能选文件

### What is actually happening?

上传文件的时候 正常应该不能选文件，但现在能选文件

| Environment | Info |
| --- | --- |
| antd | 4.24.8 |
| React | 18.2 |
| System | macOS Ventura 13.3.1 |
| Browser | safria 16.4 |

### PR DESCRIPTION
**Pull request by yoyo837**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link
close https://github.com/react-component/upload/pull/516
close https://github.com/ant-design/ant-design/issues/44170

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Add FAQ for Upload       |
| 🇨🇳 Chinese |    为Upload添加 FAQ      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 9080325</samp>

Add documentation for a workaround for Safari folder upload issue. Update `components/upload/demo/directory.md` and `components/upload/index.en-US.md` and `components/upload/index.zh-CN.md` with a link and a new section that explain how to use an invalid `accept` attribute to prevent file selection.

### <samp>🤖 Generated by Copilot at 9080325</samp>

*  Add a workaround for a Safari issue that prevents folder uploading ([link](https://github.com/ant-design/ant-design/pull/45050/files?diff=unified&w=0#diff-37e6dc09605a20a66070f496aefa182a532795cdd00ca3e969145e574663b4d7L3-R7), [link](https://github.com/ant-design/ant-design/pull/45050/files?diff=unified&w=0#diff-e13a415e44a507ae4638ea63983ee4b0b46b7910777a732251275d62e61aa0b9R162-R169), [link](https://github.com/ant-design/ant-design/pull/45050/files?diff=unified&w=0#diff-c9ad8f85ddc3c08c66b3f610bc06e0e1f8839d23a2f298c37edec8dfb18de390R162-R169))
  - Insert a link to the workaround section in the `directory.md` demo file for both English and Chinese versions ([link](https://github.com/ant-design/ant-design/pull/45050/files?diff=unified&w=0#diff-37e6dc09605a20a66070f496aefa182a532795cdd00ca3e969145e574663b4d7L3-R7))
  - Explain the workaround in the `index.en-US.md` and `index.zh-CN.md` files under the upload component documentation ([link](https://github.com/ant-design/ant-design/pull/45050/files?diff=unified&w=0#diff-e13a415e44a507ae4638ea63983ee4b0b46b7910777a732251275d62e61aa0b9R162-R169), [link](https://github.com/ant-design/ant-design/pull/45050/files?diff=unified&w=0#diff-c9ad8f85ddc3c08c66b3f610bc06e0e1f8839d23a2f298c37edec8dfb18de390R162-R169))
  - Show an example of using an invalid `accept` attribute to disable file selection when uploading a folder ([link](https://github.com/ant-design/ant-design/pull/45050/files?diff=unified&w=0#diff-e13a415e44a507ae4638ea63983ee4b0b46b7910777a732251275d62e61aa0b9R162-R169), [link](https://github.com/ant-design/ant-design/pull/45050/files?diff=unified&w=0#diff-c9ad8f85ddc3c08c66b3f610bc06e0e1f8839d23a2f298c37edec8dfb18de390R162-R169))

### CODE REVIEW

File: components/upload/index.zh-CN.md [hunk 1]
```diff
@@ -159,3 +159,11 @@ demo:
 - [#32672](https://github.com/ant-design/ant-design/issues/32672)
 - [#32913](https://github.com/ant-design/ant-design/issues/32913)
 - [#33988](https://github.com/ant-design/ant-design/issues/33988)
+
+### 文件夹上传在 Safari 仍然可以选中文件?
+
+组件内部是以 `directory`、`webkitdirectory`属性控制input来实现文件夹选择的, 但似乎在 Safari 的实现中，并不会阻止用户选择文件，请尝试额外传递无法匹配文件的 `accept` 属性来归并此问题 例如:
```
Comment (kiner-tang) [inline components/upload/index.zh-CN.md:165]: 竟然还有这种方案，哈哈

**Comment by yoyo837** [comment]
@xiaozisong

File: components/upload/index.zh-CN.md [hunk 2]
```diff
@@ -159,3 +159,11 @@ demo:
 - [#32672](https://github.com/ant-design/ant-design/issues/32672)
 - [#32913](https://github.com/ant-design/ant-design/issues/32913)
 - [#33988](https://github.com/ant-design/ant-design/issues/33988)
+
+### 文件夹上传在 Safari 仍然可以选中文件?
+
+组件内部是以 `directory`、`webkitdirectory` 属性控制input来实现文件夹选择的, 但似乎在 Safari 的实现中，并不会阻止用户选择文件，请尝试额外传递无法匹配文件的 `accept` 属性来归并此问题 例如:
```
Comment (Wxh16144) [inline components/upload/index.zh-CN.md:165]:
```suggestion
组件内部是以 `directory`、`webkitdirectory` 属性控制 input 来实现文件夹选择的, 但似乎在 Safari 的实现中，并不会阻止用户选择文件，请尝试额外传递无法匹配文件的 `accept` 属性来归并此问题 例如:
```

**Review by Wxh16144 — APPROVED** [review]

File: components/upload/index.zh-CN.md [hunk 3]
```diff
@@ -159,3 +159,11 @@ demo:
 - [#32672](https://github.com/ant-design/ant-design/issues/32672)
 - [#32913](https://github.com/ant-design/ant-design/issues/32913)
 - [#33988](https://github.com/ant-design/ant-design/issues/33988)
+
+### 文件夹上传在 Safari 仍然可以选中文件?
+
+组件内部是以 `directory`、`webkitdirectory` 属性控制 input 来实现文件夹选择的, 但似乎在 Safari 的实现中，并不会阻止用户选择文件，请尝试额外传递无法匹配文件的 `accept` 属性来归并此问题 例如:
```
Comment (afc163) [inline components/upload/index.zh-CN.md:165]:
```suggestion
组件内部是以 `directory`、`webkitdirectory` 属性控制 input 来实现文件夹选择的, 但似乎在 Safari 的实现中，[并不会阻止用户选择文件](https://stackoverflow.com/q/55649945/3040605)，请尝试额外传递无法匹配文件的 `accept` 属性来归并此问题 例如:
```

File: components/upload/index.zh-CN.md [hunk 4]
```diff
@@ -159,3 +159,11 @@ demo:
 - [#32672](https://github.com/ant-design/ant-design/issues/32672)
 - [#32913](https://github.com/ant-design/ant-design/issues/32913)
 - [#33988](https://github.com/ant-design/ant-design/issues/33988)
+
+### 文件夹上传在 Safari 仍然可以选中文件?
+
+组件内部是以 `directory`、`webkitdirectory` 属性控制 input 来实现文件夹选择的, 但似乎在 Safari 的实现中，[并不会阻止用户选择文件](https://stackoverflow.com/q/55649945/3040605)，请尝试额外传递无法匹配文件的 `accept` 属性来归并此问题 例如:
```
Comment (yoyo837) [inline components/upload/index.zh-CN.md:165]:
```suggestion
组件内部是以 `directory`、`webkitdirectory` 属性控制 input 来实现文件夹选择的, 但似乎在 Safari 的实现中，[并不会阻止用户选择文件](https://stackoverflow.com/q/55649945/3040605)，请尝试额外传递无法匹配文件的 `accept` 属性来规避此问题 例如:
```

File: components/upload/index.en-US.md [hunk 5]
```diff
@@ -159,3 +159,11 @@ Ref:
 - [#32672](https://github.com/ant-design/ant-design/issues/32672)
 - [#32913](https://github.com/ant-design/ant-design/issues/32913)
 - [#33988](https://github.com/ant-design/ant-design/issues/33988)
+
+### Can still select files when uploading a folder in Safari?
+
+Inside the upload component, we use the `directory` and `webkitdirectory` properties to control only directories can be selected. However, in Safari's implementation it doesn't seem to work. Please try passing an additional `accept` attribute that cannot match any files. For example:
```
Comment (yoyo837) [inline components/upload/index.en-US.md:165]:
```suggestion
Inside the upload component, we use the `directory` and `webkitdirectory` properties to control only directories can be selected. However, in Safari's implementation it doesn't seem to work. See [here](https://stackoverflow.com/q/55649945/3040605). Please try passing an additional `accept` attribute that cannot match any files. For example:
```

**Pull request merged**